specifications/api/swagger.yaml

```diff
@@ -1951,6 +1964,39 @@ paths:
             schema: *ApiSelectCoinsData
       responses: *responsesSelectCoins
 
+  /wallets/{walletId}/migrations:
+    get:
+      operationId: getByronShelleyMigrationInfo
+      tags: ["Shelley Migrations"]
+      summary: Calculate Cost
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Calculate the exact cost of migrating from a Shelley wallet to
+        the specified set of addresses.
```

Comment:
I have a question:

The `operationId` mentions Byron, but the description doesn't.

Is that intentional?

Reply:
Fixed